source/production/F0.Minesweeper.Components/Pages/Game/Modules/GameEndStatus.razor.cs

```diff
@@ -23,17 +23,23 @@ public GameEndStatus()
 
 		protected override void OnParametersSet()
 		{
-			EventAggregator?.GetEvent<DifficultyLevelChangedEvent>().Subscribe(OnDifficultyLevelChanged);
```

Comment:
Reverse order of unsubscription?

Reply:
See [comment in Game](https://github.com/Flash0ver/F0.Minesweeper/pull/100#discussion_r742229796)